data/seed/Seeding.md

```diff
@@ -0,0 +1,99 @@
+# Seeding the Database
+
+This covers seeding the database for development work.
+
+## Preparation
+
+You should have followed the steps for getting started and have:
+
+- PostgreSQL database and Elasticsearch search engine running in Docker containers.
+- Vite front end server running in another terminal window.
+- Django development server running in another terminal window.
+
+At this point, you should be able to login to the Admin website at
+http://localhost:8000/ or http:127.0.0.1:8000/ with the email address and
+password of the super user you created.
+
+In a fourth terminal window, have your Python virtual environment activated and
+be in your working directory with the UIPA.org source files cloned from your
+fork of the main branch of the [UIPA.org
+repository](https://github.com/CodeWithAloha/uipa).
+
+## Seeding
+
+To seed the database, run these commands:
+
+```
+$ python manage.py loaddata data/seed/new-classification.json                                
+$ python manage.py loaddata data/seed/new-jurisdiction.json                                  
```

Comment:
I got an error running this command here, do you any idea what's wrong?

```
➜  uipa git:(pr/80) ✗ python manage.py loaddata data/seed/2024-03-21-jurisdiction.json
Traceback (most recent call last):
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/psycopg/cursor.py", line 723, in execute
    raise ex.with_traceback(None)
psycopg.errors.NotNullViolation: null value in column "last_modified_at" of relation "publicbody_jurisdiction" violates not-null constraint
DETAIL:  Failing row contains (1, State of Hawaii, state-of-hawaii, , f, 1, null, null).

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/tylerchong/Desktop/workspace/codewithaloha/uipa/manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 102, in handle
    self.loaddata(fixture_labels)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 163, in loaddata
    self.load_label(fixture_label)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 253, in load_label
    if self.save_obj(obj):
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/management/commands/loaddata.py", line 209, in save_obj
    obj.save(using=self.using)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/core/serializers/base.py", line 288, in save
    models.Model.save_base(self.object, using=using, raw=True, **kwargs)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/tylerchong/.pyenv/versions/3.10.13/lib/python3.10/site-packages/psycopg/cursor.py", line 723, in execute
    raise ex.with_traceback(None)
django.db.utils.IntegrityError: Problem installing fixture '/Users/tylerchong/Desktop/workspace/codewithaloha/uipa/data/seed/2024-03-21-jurisdiction.json': Could not load publicbody.Jurisdiction(pk=1): null value in column "last_modified_at" of relation "publicbody_jurisdiction" violates not-null constraint
DETAIL:  Failing row contains (1, State of Hawaii, state-of-hawaii, , f, 1, null, null).
```

Reply:
What repo url, branch, and commit are you running this against?

Reply:
I checked out your branch with:

```
git ls-remote --refs origin
git fetch origin pull/80/head:pr/80 && git checkout pr/80
```

Commit hash `d74154ade01495f8f38f8d2578aa09ec798a51b1`

Reply:
Hmm...Did you start with a freshly initialized database?

Reply:
I was using the pre-March 6 checkout, commit d9e5322e0ed21b680f0e597997c20274f670220e, from what was `main` at that time. And I just plopped in what I created in data/seed.

I'll need to get the database running on the current main instead. Apologies!

Reply:
I just replicated the error you received. I'll need to look a bit deeper as to why it's happening. I'll be able to correct the fixture for jurisdications. I assume that other things might be broken as well so I'll address them as I find them.